labellab-flask/api/controllers/userscontroller.py

```diff
@@ -125,13 +128,83 @@ def post(self):
         # Generate the access token. This will be used as the
         # authorization header
         response = {
-            "message": "You logged in successfully.",
+            "success": True,
+            "msg": "You logged in successfully.",
             "access_token": access_token,
             "refresh_token": refresh_token,
             "user_details": to_json(user),
         }
         return make_response(jsonify(response)), 200
 
+class Auth(MethodView):
+    """
+    This class-based view handles user 
+    register and access token generation \
+    via 3rd sources like github, google
+    """
+
+    def post(self):
+        # Querying the database with requested email
+        data = request.get_json(silent=True,
+                                force=True)
+
+        try:
```

Comment:
You can use data.get('') or None. Although, this also works but if you use this method then you specifically return the response of which required field is missing.